### DISCUSSION TITLE
Give permissions to service account to trigger deploy piprlines for project in a company

### DISCUSSION THREAD

**Discussion by sfontana9** [mia-platform/community#517] [Q&A] [answered]
### What product are you having troubles with?

Console

### What Console version are you using?

13.0.4

### Description

Hello, I created a service account to act as a super user for a company.
Looking at this page in the documentation https://docs.mia-platform.eu/docs/development_suite/identity-and-access-management/console-levels-and-permission-management#console-root-level-permissions i thought the console.project.environment.deploy.trigger permission is enough to trigger the deploy pipeline, so i created my SA with this permissions

["console.root.project.create","console.root.project.details.update","console.root.project.delete", "console.root.all.view", "console.company.project.environment.deploy.trigger", "console.company.project.environment.k8s.pod.delete", "console.company.project.environment.k8s.job.delete"]

Instead, calling the api/deploy/projects/${id}/trigger/pipeline/ endpoint I get this error:  _You do not have permissions to access this feature, contact the administrator for more information_.

Am i missing something? Thanks

### Actual Outcome

The project deploy pipeline is not triggered and the response shows an error.

### Expected Outcome

The project deploy pipeline is triggered.

**Comment by fredmaggiowski** [comment, marked as answer]
Hi @sfontana9 at the moment there is no root permission to allow deploying projects, as root service account are not meant to perform such actions.

You should create a service account for each company and grant at least the **Maintainer** role
